warehouse/templates/manage/project/settings.html

```diff
@@ -351,6 +351,43 @@ <h3>
 
   <hr>
 
+  <h2>{% trans %}Archive project{% endtrans %}</h2>
+  {% set can_be_archived = not project.lifecycle_status or project.lifecycle_status == "quarantine-exit" %}
+  {% set can_be_unarchived = project.lifecycle_status == "archived" %}
+  <div class="callout-block callout-block--danger" data-controller="archive-confirm">
+  <p>
+    {% trans readme_description_href='https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/'%}
```

Comment:
Drop locale and version to let the platform auto-route
```suggestion
    {% trans readme_description_href='https://packaging.python.org/guides/making-a-pypi-friendly-readme/'%}
```